sdk/src/timing.rs

```diff
@@ -167,59 +155,4 @@ mod test {
             Duration::from_millis(1000) * ticks_per_slot
         );
     }
-
-    #[test]
-    fn test_duration_as() {
```

Comment:
FWIW - the unit test did pass after the first commit, but didn't think it was worth keeping around